style.css

```diff
@@ -471,3 +479,396 @@ h2 {
+    height: 448px;
+    width: 100%;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+    justify-items: center;
+    align-items: center;
+  }
+
+  .art-container {
+    height: 435px;
+    width: 369px;
+    margin-left: 142px;
+    background-image: url("./Images/Img-Placeholder10.png");
+  }
+
+  .art-container:hover {
+    background-size: cover;
+    background-repeat: no-repeat;
+  }
+
+  .art-container1 {
+    height: 435px;
+    width: 369px;
+    margin-left: 10px;
+    background-image: url("./Images/Img-Placeholder11.png");
+  }
+
+  .art-container2 {
+    height: 435px;
+    width: 369px;
+    margin-right: 143px;
+    background-image: url("./Images/Mask-Group.png");
+  }
+
+  .v2 {
+    padding: 12px;
+  }
+
+  .button {
+    margin-top: 49px;
+  }
+
+  .about {
+    width: 100%;
+    margin-top: 330px;
+    margin-left: 0;
+  }
+
+  .about-me-footer {
+    height: 582px;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .image-bg {
+    display: flex;
+    flex-direction: row;
+    margin-right: 14%;
+    background-image: url("./Images/illustration-about\ me@3x.png");
+    background-size: 310px 268px;
+    background-position: 10% 95%;
+    transform: rotate(360deg);
+  }
+
+  .detail {
+    display: flex;
+    flex-direction: column;
+    height: 582px;
+    width: 566px;
+    margin-left: 100px;
+  }
+
+  .about-me {
+    height: 78px;
+    width: 566px;
+    margin-left: 142px;
+    font-family: 'Crete Round', sans-serif;
+    font-size: 72px;
+    font-weight: 400;
+    line-height: 82px;
+    letter-spacing: 0;
+    text-align: left;
+  }
+
+  .footer-text {
+    font-family: 'Inter', sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 32px;
+    letter-spacing: 0;
+    text-align: left;
+    height: 148px;
+    width: 566px;
+    margin-left: 142px;
+  }
+
+  .footer-btm {
+    height: 48px;
+    width: 161px;
+    padding: 12px;
+    font-family: 'Inter', sans-serif;
+    font-size: 17px;
+    font-weight: 700;
+    line-height: 24px;
+    letter-spacing: 0.03em;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-left: 169px;
+  }
+
+  .my-skills {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    height: 596px;
+    width: 100%;
+  }
+
+  .footer-list {
+    height: 281.4558410644531px;
+    width: 289.4558410644531px;
+    padding: 0;
+  }
+
+  .footer-ul {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin-left: 100px;
+  }
+
+  .work {
+    height: 336px;
+    width: 282px;
+    left: 441.4560546875px;
+    top: 130px;
+  }
+
+  .skill {
+    height: 336px;
+    width: 282px;
+    left: 875.4560546875px;
+    top: 130px;
+  }
+
+  .contact {
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(1, 1fr);
+    margin-left: 135px;
+    margin-top: 130px;
+    height: 400px;
+  }
```

Comment:
- [ ] **[OPTIONAL]** Please make sure the content of the contact section doesn't go outside of the screen when the screen is resized. You can do that by taking advantage of flex-box features (`flex-wrap: wrap;`).

**Your project resized:**
![image](https://user-images.githubusercontent.com/74411135/236288656-ed07d643-7ce7-4e85-a4e9-66f1b50cde89.png)

This will make your project way prettier which will improve the user experience 📱  😄 👍